homeassistant/components/glances/config_flow.py

```diff
@@ -47,6 +48,49 @@ class GlancesFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a Glances config flow."""
 
     VERSION = 1
+    _reauth_entry: config_entries.ConfigEntry | None
+
+    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
+        """Perform reauth upon an API authentication error."""
+        self._reauth_entry = self.hass.config_entries.async_get_entry(
+            self.context["entry_id"]
+        )
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Confirm reauth dialog."""
+        errors = {}
+        assert self._reauth_entry
+        if user_input is not None:
+            user_input = {**self._reauth_entry.data, **user_input}
+            api = get_api(self.hass, user_input)
+            try:
+                await api.get_ha_sensor_data()
+            except GlancesApiAuthorizationError:
+                errors["base"] = "invalid_auth"
+            except GlancesApiConnectionError:
+                errors["base"] = "cannot_connect"
+            else:
+                self.hass.config_entries.async_update_entry(
+                    self._reauth_entry, data=user_input
+                )
+                await self.hass.config_entries.async_reload(self._reauth_entry.entry_id)
+                return self.async_abort(reason="reauth_successful")
+
+        return self.async_show_form(
+            description_placeholders={
+                CONF_USERNAME: self._reauth_entry.data[CONF_USERNAME]
+            },
```

Comment:
For what are the description placeholders? I miss those strings?

Reply:
Sorry, forgot to add the relevant strings.

Reply:
No problem, that's what reviews are for :)